main.py

```diff
@@ -490,7 +490,53 @@ def index(self):
             p.type = point.type
             points.append(p)
         context = {'points': points}
-        return self.render('viewhighlighted.html', **context)
+        return self.render('viewhighlighteddata.html', **context)
+
+    def is_visible(self):
+        return login.current_user.is_authenticated()
+
+class ViewHighlightedMarkersMap(BaseView):
```

Comment:
Right, we did talk. We agreed there should be a parameter to say whether to show markers or highlight points on the map, and it should go to the `url` field in the Backbone collection in `app.js`. Then the code in `index` and in the highlight point view can use a common function, passing the type of data as a parameter.

If you're doing exactly the same as `index` for now, I think you should just call `index` itself.